clickhouse/client.h

```diff
@@ -58,6 +67,14 @@ struct ClientOptions {
     /// Service port.
     DECLARE_FIELD(port, unsigned int, SetPort, 9000);
 
+    /// Hostnames of the servers. The next host to connect is selected according to the EndpointsIterationAlgorithm.
+    /// Note: If SetHost and SetHosts are setted, host will be placed at the beginning of the hosts vector.  
+    DECLARE_FIELD(hosts, std::vector<std::string>,  SetHosts, std::vector<std::string>());
+    /// Ports of the servers.
+    DECLARE_FIELD(ports, std::vector<unsigned int>, SetPorts, std::vector<unsigned int>());
+    /// Algorithm for selecting the next endpoint for connection.
+    DECLARE_FIELD(iteration_algo, EndpointsIterationAlgorithm, SetEndpointsIterationAlgorithm, EndpointsIterationAlgorithm::RoundRobin);
```

Comment:
We don't need to expose this setting at this point, since there is no meaningful way changing it to something else. It only makes sense to expose those when we have multiple algorithms.